src/main/drivers/dshot_dpwm.h

```diff
@@ -0,0 +1,162 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Author: jflyper
+ */
+
+#pragma once
+
+#define MOTOR_DSHOT1200_HZ    MHZ_TO_HZ(24)
```

Comment:
A lot of these defines seem to be for internal use in `dshot_dpwm.c` only - can we have them moved into that file please?

Reply:
Okay, here is the use statistics

`MOTOR_xSHOTnnnn_HZ` : used only in `dshot_dpwm.c`
`MOTOR_BIT_n` : used only in `dshot_dpwm.c`
`MOTOR_BITLENGTH` : used everywhere
`PROSHOT_BASE_SYMBOL` and other ProShot related constants : used everywhere

We can move HZ defs and BIT defs into `dshot_dpwm.c`, but I prefer to keep them together as is, because I need all of these to draw a timing chart.

Reply:
Fair enough.